src/components/components/ebay-notice-base/index.marko

```diff
@@ -15,11 +15,12 @@ static var ignoredAttributes = [
     "mainRoot"
 ];
 
+$ var ariaLabelledBy = input.prefixClass.includes("inline-notice") ? false : "status";
```

Comment:
Not sure it makes sense for inline notice to be part of notice base. Maybe something we can revisit in v8 refactoring.

Reply:
I think it should be, there is a lot of stuff that is being used like the icon and such.

That said, I think we should not do it this way but add a new argument or something.

Let's make a new input like `noA11yLabel`.
Also I would just do: 
`aria-labelledby=(!input.noA11yLabel && component.elId('status')`